e2e/storage_class_test.go

```diff
@@ -51,9 +98,18 @@ var _ = Describe("when Tenant handles Storage classes", func() {
 	})
 	JustAfterEach(func() {
 		Expect(k8sClient.Delete(context.TODO(), tnt)).Should(Succeed())
+		Eventually(func() (err error) {
+			err = k8sClient.DeleteAllOf(context.TODO(), &storagev1.StorageClass{}, &client.DeleteAllOfOptions{})
+			if err != nil && !k8serrors.IsNotFound(err) {
+				return err
+			}
+			return nil
```

Comment:
Let's delete all the StorageClass resources marked with the label selector `has env`, please, keeping the default one created by `kind`.